tests/Handler/ReleaseHandlerTest.php

```diff
@@ -155,6 +156,8 @@ public function setUpCommandHandler()
     /** @test */
     public function it_creates_a_new_release_for_current_branch_without_existing_tags()
     {
+        $this->git->getLastTagOnBranch()->willThrow(ProcessFailedException::class);
```

Comment:
Good catch. I actually solved it differently in #81 but this in fact the correct solution 👍 

Can you patch `expectTags` with the following?

```php
    private function expectTags(array $tags = []): void
    {
        $process = $this->prophesize(Process::class);
        $process->getOutput()->willReturn(implode("\n", $tags));

        $this->process->mustRun('git tag --list')->willReturn($process->reveal());

        if ($tags === []) {
            $this->git->getLastTagOnBranch()->willThrow(ProcessFailedException::class);
        } else {
            $this->git->getLastTagOnBranch()->willReturn(end($tags));
        }
    }
```

And revert this change https://github.com/park-manager/hubkit/pull/81/files#diff-307527f4456838fd7b83c398aefbe6aeR186 (as it's actually wrong 🤦 )

Thanks.